### DISCUSSION TITLE
Does signing into my google account affect my device limit?

### DISCUSSION THREAD

**Discussion by LukeWarmpocket** [maxrave-dev/SimpMusic#312] [Q&A]
Hello,

I subscribe to Youtube Premium, but Google has a ridiculous 10 device limit for Youtube Music. If I log into SimpMusic to bring over my playlists and such, will Google detect this as one of my youtube music devices?

Thanks.

**Comment by maxrave-dev** [comment]
I used YouTube Music Premium but I don't see where to check logged-in devices. Please show me how to

**Reply by LukeWarmpocket** [reply]
Unfortunately, there's no way to manually view that information. Back in the Google Play Music era it was possible, but now it's automated on Google's backend. Officially the limit is that you can only download videos or music on 10 devices. https://support.google.com/youtube/answer/6308288?hl=en

Would Google detect downloading music through SimpMusic as downloading through a YouTube Music client?

**Reply by maxrave-dev** [reply]
I don't know bro, try your self hic

**Comment by owenconnorz** [comment]
I wouldn't think it would as your using the same login this shouldn't be seen as a new device added

**Reply by LukeWarmpocket** [reply]
Sadly, it doesn't work that way. If you download music on a device, factory reset the device, and then download music again, google perceives this as 2 separate devices.

**Comment by maxrave-dev** [comment]
@LukeWarmpocket Hey, I confirmed I reached the premium account limit when I tested the app. So the limit is real
